src/components/RewardCards/percentage.tsx

```diff
@@ -30,12 +30,12 @@ const Percentage = styled('button')`
   border: solid 2px white;
   background: none;
 `
-interface Deposit {
+interface Percentage {
   callBack: any
   user: User
 }
 
-export default function Deposit({ user, callBack }: Deposit) {
+export default function Percentage({ user, callBack }: Percentage) {
```

Comment:
callback seems too generic as a name, can we add a name that makes it easy for someone to understand